src/app/common/hooks/use-bitcoin-contracts.ts

```diff
@@ -196,46 +191,8 @@ export function useBitcoinContracts() {
     close();
   }
 
-  async function getHealthyDlcLinkAttestor(): Promise<string> {
-    const dlcLinkAttestorUrls = [
-      'https://devnet.dlc.link/attestor-1/',
-      'https://devnet.dlc.link/attestor-2/',
-      'https://devnet.dlc.link/attestor-3/',
-    ];
-
-    let currentAttestorUrl: string | undefined;
-
-    for (const attestorURL of dlcLinkAttestorUrls) {
-      const isAttestorHealthy = await checkDlcLinkAttestorHealth(attestorURL);
-      if (isAttestorHealthy) {
-        currentAttestorUrl = attestorURL;
-        break;
-      }
-    }
-
-    if (!currentAttestorUrl) {
-      throw new Error('Unable to find a healthy DLC.Link attestor');
-    }
-
-    return currentAttestorUrl;
-  }
-
-  async function getAllSignedBitcoinContracts() {
-    let bitcoinContractInterface: JsDLCInterface | undefined;
-
-    try {
-      const currentAttestorUrl = await getHealthyDlcLinkAttestor();
-      bitcoinContractInterface = await getBitcoinContractInterface([currentAttestorUrl]);
```

Comment:
This try/catch block navigated the user to an error page. Removed the try/catch block, now it just returns early if the bitcoinContractInterface is undefined.